includes/core/class-mail.php

```diff
@@ -384,9 +384,21 @@ function prepare_template( $slug, $args = array() ) {
 			add_filter( 'um_template_tags_replaces_hook', array( &$this, 'add_replace_placeholder' ), 10, 1 );
 
 			// Convert tags in email template
-			return um_convert_tags( $message, $args );
-		}
+			$message = um_convert_tags( $message, $args );
```

Comment:
Add $context="html|plain" variable for the `um_convert_tags` for making the result more expectable and run this function with "plain" context here.